src/routes/chat.route.ts

```diff
@@ -0,0 +1,10 @@
+import { protectRoute } from "../middlewares/auth.middleware";
+import { Router } from 'express'
+import { viewAllMessage, createMessage } from "../controllers/chat.controller"; 
+
```

Comment:
Correct the path of your routes to be clear, Like "/chat/sendmessage" instead of "/post/add"

Reply:
it has already been changed